### ISSUE TITLE
[Feature Request] Option to not auto-add shortcuts to Steam on Steam Deck

### ISSUE THREAD

**Issue by eskay993** [redromnon/HeroicBashLauncher#104]
**Is your feature request related to a problem? Please describe.**
While a useful option for most people to auto-add Steam shortcuts on Steam Deck, if you also use other programs to manage adding shortcuts to Steam (like Steam ROM Manager or Boilr) then you can end up with duplicates shortcuts on Steam Deck.

**Describe the solution you'd like**
Would be great to have the ability to disable this functionality on Steam Deck so it behaves like desktop. This could possibly be added to `settings.config` as a true or false toggle (enabled by default as most people would probably want it on).

Could double up as a global settings so can toggle it on or off for any device/desktop (not just Steam Deck).

Thanks for your consideration.

**Comment by redromnon** [comment]
Sure, I can do that.

**Comment by redromnon** [comment]
Adding to this, I've only kept this setting for Deck users for the time being.

**Comment by eskay993** [comment]
Great thanks for this! Seems something is not quite right though.

First time I run it, it create a new `settings.config` as below and does **not** create a shortcut. 

```
{
  "artwork": true,
  "epic": true,
  "gog": true,
  "autoaddtosteam": true
}
```

I then change it to "autoaddtosteam": false` and it seems `settings-config` get overwritten with the default values above and it **does** create a shortcut. 

Tried changing other settings, and they are overwritten as well. Seems the file is overwritten with the default values every time HBL is run.

Let me know if there is anything else you want me to test.

**Comment by redromnon** [comment]
I'll fix this. The program first creates the file before reading it thus leading to overwriting.

**Comment by redromnon** [comment]
This commit should hopefully fix it - https://github.com/redromnon/HeroicBashLauncher/actions/runs/3099369845

**Comment by eskay993** [comment]
Tested on Steam Deck and works great now!

Only minor thing, with `"autoaddtosteam": false`, the pop up still says games have been synced to Steam (even though they haven't). So might need an extra check there.

Thanks for implementing this so quickly.

**Comment by redromnon** [comment]
Great, I'll fix that message too.

**Comment by redromnon** [comment]
Feature included in v3.0.0
